docs/how-to-guides/software-management/releases/isis-release-process.md

```diff
@@ -124,15 +178,21 @@ Clone the repo locally with git clone.
     - [ ] On the same page, create a new tag for the release version.
     - [ ] Name the release "ISISX.Y.Z LTS". 
 
+=== "Prod"
+
+    - [ ] In the release tab on GitHub, draft a release and set the target branch to the branch created in Step 4.
+    - [ ] On the same page, create a new tag for the release version.
+    - [ ] Name the release "ISISX.Y.Z Production". 
+
 !!! note ""
 
     Tag **custom** builds as **pre-release**, including ***mission-specific*** and ***non-standard*** builds.
  
 !!! success ""
 
-    Move on to step 6 after **creating the release**. 
+    Move on to step 7 after **creating the release**. 
 
-### 6. **Publish Record to DOI**
+### 7. **Publish Record to DOI**
 
 !!! info "LR/LTS only, not for RCs."
```

Comment:
Similar to previous, I think we want to also make DOIs for PROD releases so `LR/LTS/PROD`